src/fromager/commands/build.py

```diff
@@ -141,6 +141,19 @@ def _build(
     req: Requirement,
     source_download_url: str,
 ) -> pathlib.Path:
+    pbi = wkctx.package_build_info(req)
+    build_tag_from_settings = pbi.build_tag(resolved_version)
+    build_tag = build_tag_from_settings if build_tag_from_settings else (0, "")
+    build_tag_as_string = " ".join(map(str, build_tag))
+    per_wheel_logger = logging.getLogger("")
+    log_filename = (
+        req.name + " " + str(resolved_version) + "-" + build_tag_as_string + ".log"
```

Comment:
Another space in the filename?

Instead of req.name, we should use our function that converts the name to the override module format for consistency. https://github.com/python-wheel-build/fromager/blob/main/src/fromager/overrides.py#L128

Reply:
okay, done in latest push!